<ui version="4.0" >
 <class>SoftwareUpdateProgressDialog</class>
 <widget class="QDialog" name="SoftwareUpdateProgressDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Software Update</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="l_message" >
     <property name="text" >
      <string>Downloading update...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QProgressBar" name="progress" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pb_action" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
